extensions/exposition/source/HTTP/Server.ts

```diff
@@ -145,11 +145,23 @@ function supportedMethods (methods: Set<string>) {
   }
 }
 
-function negotiate (request: Request): Format | null {
+function negotiate (request: Request, message: IncomingMessage): void {
+  if (request.headers.accept !== undefined) {
+    const match = SUBTYPE.exec(request.headers.accept)
+
+    if (match !== null) {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      const { type, subtype, suffix } = match.groups!
```

Comment:
```suggestion
    if (match?.groups !== undefined) {
      const { type, subtype, suffix } = match.groups
```

Reply:
Since regex matched, we _know_ that `groups` is defined and is an object with these properties.

Reply:
Yes, the point here is to avoid eslint disable by checking the property we touching

Reply:
This is a runtime check, while the problem doesn't exist at runtime.

Reply:
I believe the correct solution here is if `match` was generic function `match<TGroups>()`, but unfortunately it's not.